File: ExamPreparation/Skeletons/Java Skeleton/src/main/java/imdb/entity/Film.java
package imdb.entity;

import javax.persistence.*;

@Entity
@Table(name = "films")
public class Film {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
	private Integer Id;

    @Column(nullable = false)
    private String Name;


    @Column(nullable = false)

    private String Genre;

    @Column(nullable = false)
    private String Director;

    @Column(nullable = false)
    private Integer Year;

    public Integer getId() {
        return Id;
    }

    public void setId(Integer id) {
        Id = id;
    }

    public String getName() {
        return Name;
    }

    public void setName(String name) {
        Name = name;
    }

    public String getGenre() {
        return Genre;
    }

    public void setGenre(String genre) {
        Genre = genre;
    }

    public String getDirector() {
        return Director;
    }

    public void setDirector(String director) {
        Director = director;
    }

    public Integer getYear() {
        return Year;
    }

    public void setYear(Integer year) {
        Year = year;
    }

}

File: ExamPreparation/Skeletons/C# Skeleton/IMDB/Models/Film.cs
using System.ComponentModel.DataAnnotations;
using System.Web.Mvc;

namespace IMDB.Models
{
    public class Film
    {
        [Key]
        public int Id { get; set; }
        [Required]
        [MinLength(1)]
        public string Name { get; set; }
        [Required]
        [MinLength(1)]
        public string Genre { get; set; }
        [Required]
        [MinLength(1)]
        public string Director { get; set; }
        [Required]
        [Range(1800,2100)]
        public string Year { get; set; }
    }
}